cfgov/cfgov/settings/base.py

```diff
@@ -147,7 +146,7 @@
 
 WAGTAIL_SITE_NAME = 'v1'
 
-SHEER_SITES = [V1_TEMPLATE_ROOT]
+SHEER_SITES = [WAGTAIL_TEMPLATE_ROOT, SHEERLIKE_TEMPLATE_ROOT]
 SHEER_ELASTICSEARCH_SERVER = os.environ.get('ES_HOST', 'localhost') + ':' + os.environ.get('ES_PORT', '9200')
```

Comment:
Because APP_DIRS forces the template engine to look in the default directory, which is /jinja2/[app-name]/ in this case, so each needs to be defined for the engine to find the templates in both contexts.